.kiro/specs/raft-storage-integration/TASK_4_5_6_COMPLETION.md

```
@@ -0,0 +1,220 @@
+cargo run --release -- --config node2.toml
+
+# 节点3（加入集群）
+cargo run --release -- --config node3.toml
+```
+
+## 参考文档
+- [实施指南](./IMPLEMENTATION_GUIDE.md)
+- [设计文档](./design.md)
+- [需求文档](./requirements.md)
+- [Adaptor README](../../src/raft/src/storage/ADAPTOR_README.md)
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)